sklearn/impute/_iterative.py

```diff
@@ -685,7 +694,7 @@ def _validate_limit(limit, limit_type, n_features):
         # IterativeImputer.estimator is not validated yet
         prefer_skip_nested_validation=False
     )
-    def fit_transform(self, X, y=None):
+    def fit_transform(self, X, y=None, **fit_params):
```

Comment:
I'd call it `params` since later we can add routing for `transform` of the subestimator as well.

Reply:
I agree.